x-pack/plugins/security_solution/public/management/common/routing.ts

```diff
@@ -81,6 +81,9 @@ export const getEndpointDetailsPath = (
   const queryParams: EndpointDetailsUrlProps = { ...rest };
 
   switch (props.name) {
+    case 'endpointDetails':
+      queryParams.show = 'details';
+      break;
```

Comment:
Do you need this?
I thought that `show=details` is the default when the `show` param is not present in the URL, so need to set it. That being said, I'm ok if we leave it (for consistency).

Reply:
I vote for consistency 😃